test/test_settings.py

```diff
@@ -1,17 +1,47 @@
 # -*- coding: utf-8 -*-
 from .helper import session
 
+import os.path as osp
+
+from mathics.core.definitions import autoload_files, Definitions
 
 def test_settings():
-    # FIXME: this is a start, but we should do more
-    for setting in (
-        "Settings`$ShowFullFormInput::usage",
-        "Settings`$ShowFullFormInput",
-        "Settings`$PygmentsStyle::usage",
-        "Settings`$PygmentsShowTokens::usage",
-        "Settings`$PygmentsShowTokens",
-        "Settings`$UseUnicode::usage",
-        "Settings`$UseUnicode",
-        "Settings`MathicsScriptVersion::usage",
-    ):
-        assert session.evaluate(setting), setting
+    import os.path as osp
+    from mathics.core.definitions import autoload_files
+    import mathicsscript
+
+    root_dir = osp.realpath(
+        osp.dirname(mathicsscript.__file__),
+    )
+
+    autoload_files(session.definitions, root_dir, "autoload")
+
+    assert session.evaluate("Settings`$ShowFullFormInput::usage").to_python() != "Settings`$ShowFullFormInput::usage"
+
+    assert type(session.evaluate("Settings`$ShowFullFormInput").to_python()) is bool
+
+    assert session.evaluate("Settings`$PygmentsStyle::usage").to_python() != "Settings`$PygmentsStyle::usage"
+
+    assert session.evaluate("Settings`$PygmentsShowTokens::usage").to_python() != "Settings`$PygmentsShowTokens::usage"
+
+    assert type(session.evaluate("Settings`$PygmentsShowTokens").to_python()) is bool
+
+    assert session.evaluate("Settings`$UseUnicode::usage").to_python() != "Settings`$UseUnicode::usage"
+
+    assert type(session.evaluate("Settings`$UseUnicode").to_python()) is bool
+
+    assert session.evaluate("Settings`MathicsScriptVersion::usage").to_python() != "Settings`MathicsScriptVersion::usage"
+
+def test_is_not_notebook():
+    import os.path as osp
+    from mathics.core.definitions import autoload_files
+
+    root_dir = osp.realpath(osp.join(
+        osp.dirname(osp.abspath(__file__)),
+        "..",
+        "mathicsscript",
+    ))
+
+    autoload_files(session.definitions, root_dir, "autoload")
```

Comment:
We probably don't want to autoload a second time.

Reply:
@rocky maybe a function `load_settings`?

Reply:
Yeah - that'd be great. About the name I think it would be good to add the word "default" or "built_in" so "load_default_settings_file"  or something like that.  

I may be over-reacting to the vagueness that is (has been) pervasive in the code using terms like "doc" and "data" and "xml" when the items in question aren't generic but mean very specific things.

Reply:
@TiagoCavalcanteTrindade BTW to get rid of the autoblack failure, run `black .`  before committing and pushing. 

Alternatively install `pre-commit` and then as part of the commit the right `black` command will get run.

Reply:
@rocky ok.